app/templates/rentals/show.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="jumbo">
+  <a href="#" class="button right up-11">Edit</a>
```

Comment:
good question.  We've been recently discussing whether to add a form section to the tutorial, and edit would be the logical place to do it.  What I think is having the one subroute is fine for now, and when we add forms to the tutorial is when we would add the edit subroute.